### PR TITLE
Mention root.cern instead of root.cern.ch

### PR DESCRIPTION
**Pull request by Axel-Naumann**
Plus missing newline at end of file.

### CODE REVIEW

**Comment by Axel-Naumann** [comment]
Indeed, that's https://sft.its.cern.ch/jira/browse/ROOT-10965

File: etc/html/HELP.html [hunk 1]
```diff
@@ -101,7 +101,7 @@ <h3><a id="module_libdep"></a>Modules' Library Dependencies</h3>
 <a href="./LibraryDependencies.html">Library Dependencies Chart</a>.
 
 <h2><a id="class"></a>Class Reference</h2>
-Pages like the one for <a href="http://root.cern.ch/root/html/TH2.html">TH2</a> are
+Pages like the one for <a href="http://root.cern/root/html/TH2.html">TH2</a> are
```
Comment (pamputt) [inline etc/html/HELP.html:104]: Link should be replaced by https://root.cern/doc/master/classTH2.html

File: etc/html/HELP.html [hunk 2]
```diff
@@ -152,7 +152,7 @@ <h3><a id="class_mem"></a>List of Data and Function Members</h3>
 them into a file.
 Because methods should have reasonable names, often the method name itself is already
 a hint on what it does.
-If you want to get a <a href="http://root.cern.ch/root/html/TNamed.html">TNamed</a>'s
+If you want to get a <a href="http://root.cern/root/html/TNamed.html">TNamed</a>'s
```
Comment (pamputt) [inline etc/html/HELP.html:155]: Link should be replaced by https://root.cern/doc/master/classTNamed.html

File: etc/html/HELP.html [hunk 3]
```diff
@@ -176,9 +176,9 @@ <h4><a id="class_mem_access"></a>Access (public / protected / private)</h4>
 
 <h4><a id="class_mem_inh"></a>Inheritance</h4>
 You can often access members of a class's base classes, just as if they are defined
-in the derived class. A <a href="http://root.cern.ch/root/html/TH1.html">histogram</a>,
+in the derived class. A <a href="http://root.cern/root/html/TH1.html">histogram</a>,
```
Comment (pamputt) [inline etc/html/HELP.html:179]: Correct link : https://root.cern/doc/master/classTH1.html

File: etc/html/HELP.html [hunk 4]
```diff
@@ -176,9 +176,9 @@ <h4><a id="class_mem_access"></a>Access (public / protected / private)</h4>
...
 for example, has a name, and you can access it using GetName() as defined in its base
-class <a href="http://root.cern.ch/root/html/TNamed.html">TNamed</a>. If you want to see
+class <a href="http://root.cern/root/html/TNamed.html">TNamed</a>. If you want to see
```
Comment (pamputt) [inline etc/html/HELP.html:181]: Correct link : https://root.cern/doc/master/classTNamed.html

File: gui/guibuilder/doc/index.txt [hunk 5]
```diff
@@ -2,17 +2,17 @@ This directory contains the GUI Builder classes.
...
  Several documents describing the I/O classes are listed below:
 <ul>
-<li><a href="ftp://root.cern.ch/root/doc/25WritingGUI.pdf">The Chapter about Writing a Graphical User Interface in the Users Guide
+<li><a href="ftp://root.cern/root/doc/25WritingGUI.pdf">The Chapter about Writing a Graphical User Interface in the Users Guide
     </a></li>
-<li><a href="ftp://root.cern.ch/root/doc/9Graphics.pdf">The Chapter about Graphics and Graphical User Interface in the Users Guide
+<li><a href="ftp://root.cern/root/doc/9Graphics.pdf">The Chapter about Graphics and Graphical User Interface in the Users Guide
     </a></li>
-<li><a href="http://root.cern.ch/drupal/content/how-use-root-gui-classes">How to use the ROOT GUI classes
+<li><a href="http://root.cern/drupal/content/how-use-root-gui-classes">How to use the ROOT GUI classes
```
Comment (pamputt) [inline gui/guibuilder/doc/index.txt:9]: https://root.cern/how-use-root-gui-classes/ is dead

File: gui/guibuilder/doc/index.txt [hunk 6]
```diff
@@ -2,17 +2,17 @@ This directory contains the GUI Builder classes.
...
     </a></li>
-<li><a href="http://root.cern.ch/drupal/content/how-use-signalslot-communication-mechanism">How to use the Signal/Slot communication mechanism
+<li><a href="http://root.cern/drupal/content/how-use-signalslot-communication-mechanism">How to use the Signal/Slot communication mechanism
```
Comment (pamputt) [inline gui/guibuilder/doc/index.txt:11]: http://root.cern/drupal/content/how-use-signalslot-communication-mechanism is dead

File: gui/guibuilder/doc/index.txt [hunk 7]
```diff
@@ -2,17 +2,17 @@ This directory contains the GUI Builder classes.
...
       </a></li>
-<li><a href="http://root.cern.ch/drupal/content/how-use-root-gui-builder">How to use the GUI Builder
+<li><a href="http://root.cern/drupal/content/how-use-root-gui-builder">How to use the GUI Builder
```
Comment (pamputt) [inline gui/guibuilder/doc/index.txt:13]: https://root.cern/how-use-root-gui-builder/ is dead

File: gui/guibuilder/doc/index.txt [hunk 8]
```diff
@@ -2,17 +2,17 @@ This directory contains the GUI Builder classes.
...
       </a></li>
-<li><a href="http://root.cern.ch/drupal/content/how-create-or-modify-style">How to Create or Modify a style
+<li><a href="http://root.cern/drupal/content/how-create-or-modify-style">How to Create or Modify a style
```
Comment (pamputt) [inline gui/guibuilder/doc/index.txt:15]: http://root.cern/drupal/content/how-create-or-modify-style is dead

File: gui/guibuilder/src/TRootGuiBuilder.cxx [hunk 9]
```diff
@@ -170,7 +170,7 @@ const char gHelpBuilder[] = "\
 \n\
 For more information, please see the GuiBuilder Howto page at:\n\
 \n\
-   http://root.cern.ch/root/HowtoGuiBuilder.html\n\
+   http://root.cern/root/HowtoGuiBuilder.html\n\
```
Comment (pamputt) [inline gui/guibuilder/src/TRootGuiBuilder.cxx:173]: http://root.cern/root/HowtoGuiBuilder.html is dead

**Review by couet — APPROVED** [review]

File: etc/html/HELP.html (hunk 1 above)
Comment (guitargeek) [reply etc/html/HELP.html:104]: Actually, http://root.cern/root/html/TH2.html also works

File: etc/html/HELP.html (hunk 2 above)
Comment (guitargeek) [reply etc/html/HELP.html:155]: http://root.cern/root/html/TNamed.html Also works

File: etc/html/HELP.html (hunk 3 above)
Comment (guitargeek) [reply etc/html/HELP.html:179]: The other one also works

File: etc/html/HELP.html (hunk 4 above)
Comment (guitargeek) [reply etc/html/HELP.html:181]: The other one also works

File: gui/guibuilder/src/TRootGuiBuilder.cxx (hunk 9 above)
Comment (guitargeek) [reply gui/guibuilder/src/TRootGuiBuilder.cxx:173]: I'll replace it with https://root.cern.ch/doc/master/group__guibuilder.html in an updated PR

File: gui/guibuilder/doc/index.txt (hunk 7 above)
Comment (guitargeek) [reply gui/guibuilder/doc/index.txt:13]: Link is already removed in ROOT master.

File: gui/guibuilder/doc/index.txt (hunk 6 above)
Comment (guitargeek) [reply gui/guibuilder/doc/index.txt:11]: Link is already removed in ROOT master.

File: gui/guibuilder/doc/index.txt (hunk 5 above)
Comment (guitargeek) [reply gui/guibuilder/doc/index.txt:9]: Link is already removed in ROOT master.

**Comment by guitargeek** [comment]
Closing this because it got replaced by another PR:
   * https://github.com/root-project/root/pull/14275

**Pull request closed without merging**